### ISSUE TITLE
gif record not working

### ISSUE THREAD

**Issue by marcosassis** [lionleaf/dwitter#511]
When I try to record a dweet I get several errors like:

    Refused to connect to 'https://bam.nr-data.net/events/1/1b1cd97153?a=99980395&v=1209.f04e2b9&to=Z1JXZ0tXWEYABkBQC14Yc0ZXVUJcDgsbXRNZQ0FWSxhQUAQBGk8NVUBGCV1BU1AVOkdRC0c%3D&rst=42833&ck=1&ref=https://www.dwitter.net/d/22873' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'connect-src' was not explicitly set, so 'default-src' is used as a fallback.

I happens both in Firefox and Chrome (Ubuntu 20.04).

**Comment by lionleaf** [comment]
You're completely right, the record dweet functionality is currently completely broken; though I don't think it's related to that error. The dweet record functionality was already super slow, so best solution might be to hide the button for now and then add a new more performant record option later on.

**Comment by Toldry** [comment]
I get a different error when trying to record:


Content-Security-Policy: The page’s settings blocked the loading of a resource (frame-src) at blob:https://dweet.dwitter.net/ed739d19-bc0d-423d-b24d-6a32c81e5ddf because it violates the following directive: “frame-src https://dweet.dwitter.net/ https://dweet.lionleaf.org/” [33807](https://www.dwitter.net/d/33807)
